inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/kafka/pom.xml

```diff
@@ -140,6 +140,10 @@
                                     <pattern>org.apache.commons.lang3</pattern>
                                     <shadedPattern>org.apache.inlong.sort.kafka.shaded.org.apache.commons.lang3</shadedPattern>
                                 </relocation>
+                                <relocation>
+                                    <pattern>org.apache.inlong.sort</pattern>
+                                    <shadedPattern>org.apache.inlong.sort.kafka.shaded.org.apache.inlong.sort</shadedPattern>
+                                </relocation>
```

Comment:
`<pattern>org.apache.inlong.sort</pattern>` and `<pattern>org.apache.inlong.sort.base</pattern>` will be effected each other?

Reply:
Fixed